chpl/chpl-api/e2e/data/chplPrefixIdentifier-test-data.json

```diff
@@ -0,0 +1,18 @@
+[
+ {
+  "chplPrefix":"CHP",
+  "identifier":"028037"
+ },
+ {
+  "chplPrefix":"CHP",
```

Comment:
Why are these two different fields? I can't think of a reason to ever use these fields in any way other than by concating them together with a dash in-between, so why not just store them that way in the first place? That would also let us use the new style format in the same way, and in the same tests, because we wouldn't have to interpret the type of identifier in each test to figure out how to compose them

Reply:
this is done